css/templates/e-commerce/page.css

```diff
@@ -493,15 +522,29 @@ amp-selector [option][selected] {
   .commerce-product-thumb {
     width: 80px;
   }
-}
 
-@media (--breakpoint-md) {
   .commerce-product-desc {
     margin-right: calc(1/12 * 100%);
   }
 }
 
 
+/* BLOG */
+.commerce-blog-wrapper {
+  background-color: var(--color-white);
+}
+
+@media (--breakpoint-md) {
+  .commerce-blog {
+    margin-left: calc(1/12 * 100%);
```

Comment:
I'm using this in 3 places now. Should I make it a css-var or make a utility class?